### PR TITLE
131 add next question button for host

### LINKED ISSUE #131: Add next question button for host

### PR DESCRIPTION
**Pull request by ivarhe**
closes #131

### CODE REVIEW

**Review by MatiasGranHenriksen — APPROVED** [review]
Very nice

File: web/app/routes/start/index.tsx [hunk 1]
```diff
@@ -21,6 +22,12 @@ export default function StartQuizIndexRoute() {
 
     const startQuiz = async (quizId: number | undefined) => {
         // @ts-ignore
+        fetcher(`${window.env.API_URL}/quiz/${quizId}`)
```
Comment (Henrhh) [inline web/app/routes/start/index.tsx:25]: legg window.env greier i en egen variabel så er dere fri // @ts-ignore hele tia:)

**Review by Henrhh — APPROVED** [review]

File: web/app/routes/start/index.tsx (hunk 1 above)
Comment (perkynades) [reply web/app/routes/start/index.tsx:25]: Hold your horses for a bit, this is a remix issue

Comment (ivarhe) [reply web/app/routes/start/index.tsx:25]: How long should i hold my horses @perkynades? They are getting pretty impatient 🏇 💨

**Pull request merged**